src/info/code_size.rs

```diff
@@ -9,30 +8,30 @@ use crate::{
 
 #[derive(Serialize)]
 #[serde(rename_all = "camelCase")]
-pub struct LocInfo {
-    pub lines_of_code: usize,
+pub struct CodeSizeInfo {
+    pub size: usize,
```

Comment:
Can you explain what is the difference between this `size` (which replaces loc) and the other already existing `size` info line.

Reply:
Yeah, this is one of the things that I wanted to discuss since, as it is, it's going to be really confusing to see two size stats for the typical user. But `CodeSizeInfo` is the total size of *blobs at `HEAD`.* E.g. "how big is `src/*` for the latest commit?" `SizeInfo` is, per my understanding, *all* entries, so that would include things like commit messages. @Byron could probably explain it better.

But if you, for example, delete all your code, then "code size" will drop much more significantly than "repo size." And "code size" will be limited to *detected* code, obviously.

I guess a simple example would be that an assets repository, like a collection of images, could have a very large repository size, but a small code size.

Off-topic: regarding "assets" repositories, it might be nice to have an "unknown" ASCII image for repositories that don't have any source code.

Reply:
Feels like we might want to drop this info line alltogether 🤔

Reply:
In which case, maybe `gengo` could directly output the percentage per language instead of the `size`

Reply:
> In which case, maybe `gengo` could directly output the percentage per language instead of the `size`

Sounds good! I was actually originally going to do that, and that's what `gengo::Analysis::Summary` is for. But once I saw that `Vec<(Language, f64)>` was used in so many places, I opted for getting this is a compilable, reviewable state. I'll look into this, and probably refactoring the code, when I have a good opportunity to dive in.

Reply:
Sorry, think I misread the last comment. I think it makes sense to stop passing around `Vec<(Language, f64)>` everywhere and look into simplifying/refactoring the code. Even though a big inspiration for gengo was improving onefetch's language detection, I don't necessarily want to make it a dedicated "onefetch support" crate, and would prefer more generic implementations, like returning sizes instead of percentages.